### PR TITLE
ALT-434: Refactor load test

### PR DESCRIPTION
**Pull request by nviktorov**
https://track.veriblock.org/issue/ALT-434

### CODE REVIEW

File: pypoptools/pypoptesting/framework/generate_util.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import itertools
+import random
+import time
+
+from .random_miner import RandomPopMiner
+
+
+def generate_endorsed(node, mock_miner, atvs, vtbs, alt_blocks, vbk_blocks, btc_blocks, seed, timeout=60):
```
Comment (Warchant) [inline pypoptools/pypoptesting/framework/generate_util.py:8]: use types please
```
atvs: int, vtbs: int, ...
```

File: pypoptools/pypoptesting/framework/generate_util.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    )
+
+    start = time.time()
+    for command in commands:
+        if time.time() - start >= timeout:
+            break
+        command()
+
+
+def parentheses_sequence(open_object, close_object, n):
```
Comment (Warchant) [inline pypoptools/pypoptesting/framework/generate_util.py:38]: please add either link to algo description, or a comment

**Review by Warchant — APPROVED** [review]
Add a comment and you can merge

File: pypoptools/pypoptesting/framework/generate_util.py (hunk 1 above)
Comment (nviktorov) [reply pypoptools/pypoptesting/framework/generate_util.py]: fixed

File: pypoptools/pypoptesting/framework/generate_util.py (hunk 2 above)
Comment (nviktorov) [reply pypoptools/pypoptesting/framework/generate_util.py]: fixed

**Review by Warchant — APPROVED** [review]

**Pull request merged**